src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -118,7 +118,7 @@
         - name: --outbound-type
           type: string
           short-summary: How outbound traffic will be configured for a cluster.
-          long-summary: Select between loadBalancer and userDefinedRouting. If not set, defaults to type loadBalancer. Requires --network-plugin to be azure and a standard load balancer SKU.
+          long-summary: Select between loadBalancer and userDefinedRouting. If not set, defaults to type loadBalancer. Requires --vnet-subnet-id to be provided with a preconfigured route table and --load-balancer-sku to be Standard.
```

Comment:
seems `--vnet-subnet-id` is an existing param. it's not a good name, normally we jsut call it `--subnet-id` in cli. anyway, not related to this PR direclty

Reply:
Regardless, I appreciate the advice. If I have a free cycle I will try and send a PR to clean it up